lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -46,14 +48,29 @@ def setUp(self):
 
     def verify_access(self, mock_unit, student_should_have_access):
         """ Verify the expected result from _has_access_descriptor """
+
         self.assertEqual(
             student_should_have_access,
-            access._has_access_descriptor(self.anonymous_user, 'load', mock_unit, course_key=self.course.course_key)
+            bool(access._has_access_descriptor(
+                self.anonymous_user, 'load', mock_unit, course_key=self.course.course_key))
```

Comment:
You may get a quality error here.  The closing parenthesis has to match the indentation of its corresponding one.